baybe/recommenders/pure/nonpredictive/sampling.py

```diff
@@ -52,13 +54,19 @@ def __str__(self) -> str:
         return to_string(self.__class__.__name__, *fields)
 
 
+@define
```

Comment:
```suggestion
@define(kw_only=True)
```

not very consistent in our code yet but I think we should follow this since its also been done for `BotorchRecommender` (and should be done for all other recommenders @AdrianSosic ?)